tools/getWebpackConfig.js

```diff
@@ -56,10 +62,10 @@ module.exports = function getWebpackConfig(options) {
   };
 
   if (enableCssModules) {
-    config.resolve.extensions.push(".css");
+    config.resolve.extensions = [...config.resolve.extensions, ".css"];
 
     config.module.rules.push({
-      test: /\.css$/,
+      test: /\.module.css$/,
```

Comment:
@fahad19 changed the css module extention to `.module.css`, do you think it's going to break existing css modules in other workspaces of the monorepo?

Reply:
From a quick look I don't see any other `.css` files except of `site`.

Reply:
if `site` package works, it should be alright. this is the only place with CSS.